nixos/modules/config/update-users-groups.pl

```diff
@@ -213,7 +213,15 @@ sub parseUser {
 
     # Create a home directory.
     if ($u->{createHome}) {
-        make_path($u->{home}, { mode => 0700 }) if ! -e $u->{home};
+        my $new_home = 0;
+        if ( !-e $u->{home} ) {
+            make_path($u->{home}, { mode => 0700 });
+            $new_home = 1;
+        }
+        if (defined $new_home && $u->{initialHomeContents}) {
+            system("cp --recursive --preserve=mode --no-preserve=owner $u->{initialHomeContents}/. $u->{home}");
+            system("chown -R $u->{uid}, $u->{gid}, $u->{home}");
```

Comment:
It seems like this is throwing an error for me when testing because of the commas.
![screenshot_20180612_084212](https://user-images.githubusercontent.com/6174343/41291017-8656c516-6e1c-11e8-8cf6-d9cf4c4ebccb.png)